public/famous/main.js

```diff
@@ -1,408 +0,0 @@
-define(function(require, exports, module) {
-
-Metamaps.Famous = {};
-Metamaps.Famous.build = function () {
-    // import dependencies
-    var Engine = require('famous/core/Engine');
-    var Modifier = require('famous/core/Modifier');
-    var Transform = require('famous/core/Transform');
-    var Surface = require('famous/core/Surface');
-    var Timer = require('famous/utilities/Timer');
-    var Scrollview = require('famous/views/Scrollview');
-    var ContainerSurface = require('famous/surfaces/ContainerSurface');
-    var RenderNode = require('famous/core/RenderNode');
-
```

Comment:
This file feels like the most "JUST MAKE IT WORK" file of all time. I can't believe all of this crazy code has been running Metamaps for 1.5 years